docs/source/github/github.rst

```diff
@@ -267,26 +267,26 @@ You can use the GitHub cherry-pick command, or an easier option is to paste the
 
 .. code::
 
-   @Mergifyio backport release-2.0
+   @Mergifyio backport release-2.2
 
 Replace ``2.0`` with the branch that you want to backport to. If there are no merge conflicts,
 a new PR is automatically generated in that branch that still requires the normal approval process to be merged.
 Remember to add a comment to the original PR for each branch that you want to backport to.
 
 If there are merge conflicts, use the GitHub ``cherry-pick`` command instead, by providing the ``SHA`` from the commit in the main branch.
 
-- The following example shows how to cherry-pick a commit from the main branch into the release-2.0 branch:
+- The following example shows how to cherry-pick a commit from the main branch into the release-2.2 branch:
```

Comment:
@denyeart What do you think about updating these instructions to `2.2`?

Reply:
It is just an example, it is not something we want to maintain every release.

Reply:
Very well